packages/gui/src/components/app/AppState.tsx

```diff
@@ -148,6 +153,21 @@ export default function AppState(props: Props) {
     );
   }
 
+  if ((backendVersion && version) && (versionDialog === true)) {
+    const guiVersion = version.replace('-','.');
```

Comment:
Sorry for the back-and-forth on this. After testing this using install-gui.sh, packages/gui/package.json has version set to something like: `"1.7.0-b2.dev7"` which is transformed into `"1.7.0.b2.dev7"` by the above code. `chia version` reports `1.7.0b2.dev7`, so really we should just drop the `-`

```suggestion
    const guiVersion = version.replace('-','');
```